tests/integration/master.test.js

```diff
@@ -0,0 +1,255 @@
+  }, 20000);
+
+  test('Root call with invalid votes', async () => {
+    let key = u8a_hex(randomish_u8a());
+    let val = u8a_hex(randomish_u8a());
+    const did_to_key = [
+      [ALICE_DID, await keypair(randomish_u8a())],
+      [CHARLIE_DID, await keypair(randomish_u8a())],
+    ];
+    await master_set_storage(nc, key, val, did_to_key);
+    let sto = await nc.rpc.state.getStorage(key);
+    assert(sto.isNone, "storage item should not have been set");
+  }, 20000);
+
+  test('Root call with valid votes', async () => {
+    let key = u8a_hex(randomish_u8a());
+    let val = u8a_hex(randomish_u8a());
+    const did_to_key = [
+      [ALICE_DID, await keypair(ALICE_SK)],
+      [CHARLIE_DID, await keypair(CHARLIE_SK)],
+    ];
+    await master_set_storage(nc, key, val, did_to_key);
+    let sto = await nc.rpc.state.getStorage(key);
+    let u8a = sto.unwrap();
+    expect(u8a_hex(u8a)).toEqual(val);
+  }, 20000);
+
+  test('Root call with valid votes but insufficient vote count', async () => {
+    let key = u8a_hex(randomish_u8a());
+    let val = u8a_hex(randomish_u8a());
+    const did_to_key = [
+      [ALICE_DID, await keypair(ALICE_SK)]
+    ];
+    await master_set_storage(nc, key, val, did_to_key);
+    let sto = await nc.rpc.state.getStorage(key);
+    assert(sto.isNone, "storage item should not have been set");
+  }, 20000);
+
+  test('Root call with valid votes and oversufficient vote count', async () => {
+    let key = u8a_hex(randomish_u8a());
+    let val = u8a_hex(randomish_u8a());
+    const did_to_key = [
+      [ALICE_DID, await keypair(ALICE_SK)],
+      [BOB_DID, await keypair(BOB_SK)],
+      [CHARLIE_DID, await keypair(CHARLIE_SK)],
+    ];
+    await master_set_storage(nc, key, val, did_to_key);
+    let sto = await nc.rpc.state.getStorage(key);
+    let u8a = sto.unwrap();
+    expect(u8a_hex(u8a)).toEqual(val);
+  }, 20000);
+
+  test('Root call with votes not sorted lexically', async () => {
+    let key = u8a_hex(randomish_u8a());
+    let val = u8a_hex(randomish_u8a());
+    const did_to_key = [
+      [BOB_DID, await keypair(BOB_SK)],
+      [ALICE_DID, await keypair(ALICE_SK)],
+    ];
+    await master_set_storage(nc, key, val, did_to_key);
+    let sto = await nc.rpc.state.getStorage(key);
+    let u8a = sto.unwrap();
+    expect(u8a_hex(u8a)).toEqual(val);
+  }, 20000);
+});
+
+// connect to running node
+async function connect() {
+  const extraTypes = {
+    Address: 'AccountId',
+    LookupSource: 'AccountId',
+  };
+  return await ApiPromise.create({
+    provider: new WsProvider(FullNodeEndpoint),
+    types: {
+      ...types,
+      ...extraTypes,
+    },
+  });
+}
+
+// load a DID kp from secret
+async function keypair(seed) {
+  assert(seed instanceof Uint8Array, "wrong type");
+  await cryptoWaitReady();
+  let keyring = new Keyring({ type: 'sr25519' });
+  let key = keyring.addFromSeed(seed);
+  return key
+}
+
+async function sign_send_tx(extrinsic) {
```

Comment:
This function is a duplicate of `signAndSend` from DockAPI object. The following lines should do what this function does
```
// dock is DockAPI object
dock.setAccount(key);
dock.signAndSend(extrinsic)
```
The naming should follow camelCase.

Reply:
Same comment as above.

Reply:
Switched to camelCase in 974bafd.